plugins/nodes/src/js/columns/NodesTableMemBarColumn.tsx

```diff
@@ -3,12 +3,18 @@ import Node from "#SRC/js/structs/Node";
 // TODO: DCOS-39079
 // import { IWidthArgs as WidthArgs } from "@dcos/ui-kit/packages/table/components/Column";
 import { IWidthArgs as WidthArgs } from "#PLUGINS/nodes/src/js/types/IWidthArgs";
+import ProgressBar from "#SRC/js/components/ProgressBar";
 
 export function membarRenderer(data: Node): React.ReactNode {
-  // TODO: DCOS-38823
-  return <span>{data.get("used_resources").mem.toString()}</span>;
+  var value = data.get("used_resources").mem.toString();
+
+  return <ProgressBar
+  className="hidden-medium-down"
```

Comment:
this isnt applicable anymore (just remove it) - there is a Issue linked in the CPU bar, can you please add is as comment here? 🙇